src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -330,6 +330,21 @@ public void timeEvent(final String eventName) {
     }
 
     /**
+     * Map version of the track API below.
+     *
+     * @param eventName The name of the event to send.
+     * @param properties A Map containing the key value pairs of the properties to include in this event.
+     *                   Pass null if no extra properties exist. Cannot be null.
+     */
+    public void trackMap(String eventName, Map<String, Object> properties) {
+        if (null == properties) {
```

Comment:
I do believe we need to handle null here since the JSONObject constructor doesn't handle it
![image](https://cloud.githubusercontent.com/assets/7267883/7478331/eab1f8a0-f30e-11e4-951c-ab778ca44ffd.png)